OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -349,4 +349,4 @@
   </Target>
   -->
   <ItemGroup />
-</Project>
+</Project>
```

Comment:
You should stage just the ```<Compile Include="Traits\CreatesDisruptionField.cs" />``` line. We're trying to avoid staging other lines the different IDEs keep automatically changing so they don't go back and forth all the time.